prov/efa/src/rdm/rxr_pkt_type_misc.c

```diff
@@ -184,12 +200,16 @@ void rxr_pkt_handle_handshake_recv(struct rxr_ep *ep,
 	 */
 	peer->nextra_p3 = handshake_pkt->nextra_p3;
 	memcpy(peer->extra_info, handshake_pkt->extra_info,
-	       (handshake_pkt->nextra_p3 - 3) * sizeof(uint64_t));
+		   (handshake_pkt->nextra_p3 - 3) * sizeof(uint64_t));
```

Comment:
why changing the alignment here?

Reply:
I reformatted this function. I appears git shows a different tab size.